test/exsom_test.exs

```diff
@@ -5,7 +5,7 @@ defmodule ExsomTest do
   @test_xml ~S[<foo attr="yo"><bar>1</bar></foo>]
 
 
-  test "integration / model -> instance -> write (file)" do
+  test "integration / model (file) -> instance -> write" do
```

Comment:
We should probably rename that `Exsom.write` function, it's confusing. It's actually just creating a XML document.